docs/MessageInput.md

```diff
@@ -238,154 +236,88 @@ Create your custom Message input layout named `view_custom_message_input` as sho
 </layout>
 ```
 
-#### Step 2: Create `CustomMessageInputView`
+#### Step 2: Create CustomMessageInputView
 
-As a next step we need to connect your message input to the rest of the UI.
-There a few things to think about:
+As a next step we need to extend `MessageInputView` so you can use all features by extending `MessageInputView` simply.
```

Comment:
@adrian09h  this is not a valid english sentence :)

Reply:
@adrian09h don't you have to mention that a MessageInputView must implement the `MessageInputManager` interface?